agent/src/softwarecontaineragent.cpp

```diff
@@ -1,14 +1,34 @@
+/*
+ * Copyright (C) 2016 Pelagicore AB
+ *
+ * Permission to use, copy, modify, and/or distribute this software for
+ * any purpose with or without fee is hereby granted, provided that the
+ * above copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+ * WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+ * BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+ * OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+ * WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+ * ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+ * SOFTWARE.
+ *
+ * For further information see LICENSE
+ */
+
 #include "softwarecontaineragent.h"
 #include <cstdint>
 
 SoftwareContainerAgent::SoftwareContainerAgent(
-        Glib::RefPtr<Glib::MainContext> mainLoopContext
-        , int preloadCount
-        , bool shutdownContainers
-        , int shutdownTimeout)
-: m_mainLoopContext(mainLoopContext)
-, m_preloadCount(preloadCount)
-, m_shutdownContainers(shutdownContainers)
+    Glib::RefPtr<Glib::MainContext> mainLoopContext,
+    int preloadCount,
+    bool shutdownContainers,
+    int shutdownTimeout,
+    const std::string &configPath)
+    : m_mainLoopContext(mainLoopContext)
+    , m_preloadCount(preloadCount)
+    , m_shutdownContainers(shutdownContainers)
```

Comment:
This indentation makes it difficult to see the difference between the parameters and the initialization of members.